### PR TITLE
docker: add libtool and automake to alpine

### PR DESCRIPTION
**Pull request by maclover7**
Fixes: https://github.com/nodejs/build/issues/1186

Rolling this out now across the cluster...

- [x] test-digitalocean-ubuntu1604_docker-x64-1
- [x] test-digitalocean-ubuntu1604_docker-x64-2
- [x] test-joyent-ubuntu1604_docker-x64-1
- [x] test-softlayer-ubuntu1604_docker-x64-1

cc @santigimeno @bnoordhuis @cjihrig

### CODE REVIEW

File: ansible/roles/docker/templates/alpine36.Dockerfile.j2 [hunk 1]
```diff
@@ -37,7 +37,10 @@ RUN apk add --no-cache \
         procps \
         openssh-client \
         py2-pip \
-        bash
+        bash \
+        automake \
+        libtool \
+        m4
```
Comment (bnoordhuis) [inline ansible/roles/docker/templates/alpine36.Dockerfile.j2]: Can you install autoconf instead?  That also pulls in m4.

**Comment by maclover7** [comment]
Looks like relevant sub builds for libuv-test-commit-linux are now passing: https://ci.nodejs.org/job/libuv-test-commit-linux/906/

**Comment by maclover7** [comment]
@bnoordhuis @cjihrig Please let me know if there are any other changes you would like here, or else will go ahead and land

**Comment by maclover7** [comment]
Landing...

**Comment by maclover7** [comment]
Landed in fa7e316955f01ba18d2eba1f6f41dedb2dc31e6a

**Comment by maclover7** [comment]
@bnoordhuis Thought I had run the playbook on all of the machines, but just re-ran it, hopefully should be good now

**Comment by bnoordhuis** [comment]
Thanks, looks like it's working now.

**Pull request merged**